dags.yaml

```diff
@@ -1042,3 +1042,20 @@ bqetl_status_check:
   schedule_interval: "1h"
   tags:
     - impact/tier_1
+
+bqetl_download_funnel_attribution:
+  description: Daily aggregations of data exported from Google Analytics joined with Firefox download data.
+  default_args:
+    depends_on_past: false
+    email:
+      - gleonard@mozilla.com
+      - telemetry-alerts@mozilla.com
+    end_date: null
+    owner: gleonard@mozilla.com
+    retries: 2
+    retry_delay: 30m
+    start_date: '2023-04-10'
+  schedule_interval: 0 23 * * *
+  tags:
+    - impact/tier_1
+    - repo/bigquery-etl
```

Comment:
```suggestion
```

if I recall correctly, for the DAGs inside the bqetl repo this tag gets added automatically during the dag generation process.

Reply:
I followed the instructions here https://mozilla.github.io/bigquery-etl/cookbooks/common_workflows/ to create a new DAG file.  You're suggesting that I delete that tag and it will be re-added when the DAG file is deployed to Airflow?

Reply:
The tag will be added to your DAG definition when you run the `./bqetl dag generate` command

Reply:
I honestly can't remember.  Is there an issue with leaving it as is?